.ci/test_r_package.sh

```diff
@@ -35,6 +35,10 @@ fi
 # Installing R precompiled for Mac OS 10.11 or higher
 if [[ $OS_NAME == "macos" ]]; then
 
+    # temp fix for basictex
+    if [[ $AZURE == "true" ]]; then
+        brew update
+    fi
```

Comment:
Why is this change only for AZURE? just to save build time?

ugh the Azure Mac builds are such a pain. I had to do another "if Azure" thing on a PR I'm working on: 

![image](https://user-images.githubusercontent.com/7608904/79160038-c1537b80-7d9e-11ea-80b1-47e3cfd65c07.png)

Reply:
> Why is this change only for AZURE? just to save build time?

Yeah! Seems that Travis does it automatically during `setup.sh`.

Reply:
I saw those exact warnings actually. Thanks for the reminder!

Still just experimenting with things 😬